test/allow.js

```diff
@@ -0,0 +1,131 @@
+const assert = require('chai').assert
+const { getUserId, verifyDelegator } = require('../lib/handlers/allow')
+const rdf = require('rdflib')
+const ns = require('solid-namespace')(rdf)
+
+const alice = 'https://alice.com'  // principal
+const agentWebId = 'https://agent.example.com'  // secretary
+
+const verifiedSecretaryGraph = rdf.graph()
+verifiedSecretaryGraph.add(
+    rdf.namedNode(alice),
+    ns.acl('delegates'),
+    rdf.namedNode(agentWebId)
+  )
+const notVerifiedSecretaryGraph = rdf.graph()
+
+describe('allow() handler', () => {
+  describe('getUserId()', () => {
```

Comment:
I expected there to be two separate (not nested) describe blocks - one for the `allow` function and one for the `getUserId()` function - since they are separate exported functions from the `allow` module.